docs/environment-and-software/vscode.md

```diff
@@ -0,0 +1,197 @@
+# Visual Studio Code
+
+Visual Studio Code, also commonly referred to as VS Code, is a source-code editor developed by Microsoft for Windows, Linux, macOS and web browsers. Features include support for debugging, syntax highlighting, intelligent code completion, snippets, code refactoring, and embedded version control with Git.
+
+The Visual Studio Code Remote - SSH extension allows you to open a remote folder on any remote machine, virtual machine, or container with a running SSH server and take full advantage of VS Code's feature set. Once connected to a server, you can interact with files and folders anywhere on the remote filesystem.
+
+VSCode is available for NCAR issued laptops in the *Self Service* application or can be [downloaded](https://code.visualstudio.com/download) directly from the Microsoft VS Code website.
+
+## Connecting to Derecho or Casper
+
+The Visual Studio Code Remote SSH extension allows you to connect to a NCAR system login node and to the GLADE filesystem. Once connected to a server, you can interact with files and folders anywhere on GLADE.  You can connect to the Derecho or Casper login nodes by following these steps:
```

Comment:
I would suggest changing the terminology from "NCAR system login node" to "NCAR HPC systems".

Reply:
> I would suggest changing the terminology from "NCAR system login node" to "NCAR HPC systems".

or something with less emphasis on login nodes. We can use "Derecho and Casper" instead.

Reply:
I am marking this as resolved since this has been addressed by the changes you include here.